src/database/stores/CassandraDbDumpStore.ts

```diff
@@ -28,7 +28,7 @@ export class CassandraDbDumpStore implements DumpStore {
     }
 
     public async get(id: string): Promise<Dump | undefined> {
-        const dump = await this.cassandra.execute('SELECT id, content, embeds, channelid, TTL(content) as expiry FROM message_outputs WHERE id = :id', { id }, { prepare: true });
+        const dump = await this.cassandra.execute('SELECT id, content, embeds, channelid, TTL(channelid) as expiry FROM message_outputs WHERE id = :id', { id }, { prepare: true });
```

Comment:
what is the impact on existing dumps?

Reply:
None, the TTL for all columns is the same, its just that content can be null so the TTL is null too